react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -161,15 +188,19 @@ const styles = theme => {
             flexWrap: 'nowrap',
             alignItems: 'center',
             justifyContent: 'center',
-            backgroundColor: 'rgba(0, 0, 0, .6)',
+            backgroundColor: 'rgba(51, 51, 51, .5)',
```

Comment:
This color is repeated several times. Can you pl add a token for it?

Reply:
Discussed it with Tibi and we don't want a token for it as it's just used here. So I just extracted it into a constant